### PR TITLE
feat: added default alert case for failing replication

### PR DESCRIPTION
**Pull request by Lennart01**
Hey,
as long as #2800 is in the wild I thought it would be a good idea to add a default alert case to warn the user if this edge case occurs.
The alert was tested against a broken and a working cluster and performed as expected (also tested with single replica cluster)

### CODE REVIEW

File: docs/src/samples/monitoring/alerts.yaml [hunk 1]
```diff
@@ -55,3 +55,12 @@ groups:
     for: 1m
     labels:
       severity: warning
+  - alert: ReplicaFailingReplication
+    annotations:
+      description: Replica {{ $labels.pod }} is failing to replicate
+      summary: Checks if the replica is failing to replicate
+    expr: |-
+      (1 - cnpg_pg_replication_in_recovery + cnpg_pg_replication_is_wal_receiver_up) < 1
```
Comment (gbartolini) [inline docs/src/samples/monitoring/alerts.yaml:63]: Not familiar with Prometheus alerts, but why not:

```
cnpg_pg_replication_in_recovery > cnpg_pg_replication_is_wal_receiver_up
```

Comment (Lennart01) [reply docs/src/samples/monitoring/alerts.yaml:63]: Yup, works will change accordingly 👍 .

**Review by phisco — APPROVED** [review]

**Comment by phisco** [comment]
@Lennart01, looks like the DCO step is failing, can you fix that? see https://github.com/cloudnative-pg/cloudnative-pg/pull/2869/checks?check_run_id=17206660410 for more details 🙏

**Comment by Lennart01** [comment]
@phisco Sorry about that. Should be fixed 👍

**Review by leonardoce — APPROVED** [review]

**Pull request merged**